sdk/eventgrid/azure-eventgrid/tests/test_eg_consumer.py

```diff
@@ -0,0 +1,37 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import logging
+import sys
+import os
+import pytest
+
+from devtools_testutils import AzureMgmtTestCase, ResourceGroupPreparer
+
+from azure.core.credentials import AzureKeyCredential
+from azure.eventgrid import EventGridPublisherClient, CloudEvent, EventGridEvent
+
+from eventgrid_preparer import (
+    EventGridTopicPreparer
+)
+
+
+class EventGridConsumerTests(AzureMgmtTestCase):
+    pass
+    #@pytest.mark.liveTest
```

Comment:
is this deprecated?  or temporarily disabled?

Reply:
disabled while I run publisher tests, will add more after

Reply:
Use @pytest.mark.skip for a slightly more concise way of disabling a test :)